CODEOWNERS

```diff
@@ -5,12 +5,12 @@
 # https://git-scm.com/docs/gitignore#_pattern_format
 #
 
+* @azure/azure-functions-core
+
 # Adding codeowner for Python specific files such that GitHub automatically adds python folks as a reviewer.
 build/python.props @vrdmr @gavin-aguiar @YunchuWang
 
 # Deps.json validation file
 test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json @fabiocav @brettsam @mathewc
```

Comment:
You should also add @azure/azure-functions-core to this list as well

Reply:
To what list? `test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json` ? The list here is restricting to folks with context on what is required to change when this file is touched.

Reply:
We just had a sync about this PR. Double checking with @fabiocav and @liliankasem if we need `@azure/azure-functions-core` for deps.json file?

Reply:
I don't believe so, as what we have here is to also help folks in @azure/azure-functions-core, but want to make sure I'm not missing anything based on the question.